File: swagger/models/pagination_response.py
from .base_model import BaseModel


class PaginationResponse(BaseModel):
	"""
	no description available

	Attributes:

		requested_limit
		Limit from pagination request

		requested_offset
		Offset from pagination request

		page_size
		Number of results returned in this response

		total_results
		Total number of results in dataset
	"""

	input_map = {
		'RequestedLimit': 'requested_limit',
		'RequestedOffset': 'requested_offset',
		'PageSize': 'page_size',
		'TotalResults': 'total_results',
		}

	output_map = {
		'requested_limit': 'RequestedLimit',
		'requested_offset': 'RequestedOffset',
		'page_size': 'PageSize',
		'total_results': 'TotalResults',
		}

File: swagger/Models/PaginationResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class PaginationResponse extends BaseModel
{
	/**
	 * Limit from pagination request
	 * 
	 * @var integer
	 */
	public $requestedLimit;
	/**
	 * Offset from pagination request
	 * 
	 * @var integer
	 */
	public $requestedOffset;
	/**
	 * Number of results returned in this response
	 * 
	 * @var integer
	 */
	public $pageSize;
	/**
	 * Total number of results in dataset
	 * 
	 * @var integer
	 */
	public $totalResults;
}
